features/invite/impl/src/main/kotlin/io/element/android/features/invite/impl/response/InvalidDataException.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2025 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+package io.element.android.features.invite.impl.response
+
+class InvalidDataException : Exception()
```

Comment:
This is not used to render specific error, is it expected? I guess this is only to support very edge cases?